src/include/migraphx/streamutils.hpp

```diff
@@ -61,34 +62,50 @@ inline stream_range_container<Range> stream_range(const Range& r)
 namespace detail {
 
 template <class T>
-auto stream_write_value_impl(rank<1>, std::ostream& os, const T& x) -> decltype(os << x, void())
+auto stream_write_value_impl(rank<2>, std::ostream& os, const T& x) -> decltype(os << x, void())
 {
     os << x;
 }
 
 template <class T>
-void stream_write_value_impl(rank<1>, std::ostream& os, const std::vector<T>& r)
+void stream_write_value_impl(rank<2>, std::ostream& os, const std::vector<T>& r)
 {
     os << "{";
     os << stream_range(r);
     os << "}";
 }
 
 template <class Range>
-auto stream_write_value_impl(rank<0>, std::ostream& os, const Range& r)
+auto stream_write_value_impl(rank<1>, std::ostream& os, const Range& r)
     -> decltype(r.begin(), r.end(), void())
 {
     os << "{";
     os << stream_range(r);
     os << "}";
 }
 
+template <class T>
```

Comment:
This should do `MIGRAPHX_REQUIRES(is_reflectable<T>{})`

Reply:
That is `template <class T, MIGRAPHX_REQUIRES(is_reflectable<T>{})>`

Reply:
Done